infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -167,16 +160,13 @@ public void trim(LogAddress logAddress) {
                 handle.getPendingTrims().contains(logAddress.getAddress())) {
             return;
         }
-
         TrimEntry entry = TrimEntry.newBuilder()
                 .setChecksum(getChecksum(logAddress.getAddress()))
                 .setAddress(logAddress.getAddress())
                 .build();
 
         // TODO(Maithem) possibly move this to SegmentHandle. Do we need to close and flush?
-        OutputStream outputStream = Channels.newOutputStream(handle.getPendingTrimChannel());
```

Comment:
Ok, ok, closing this will close the parent channel. But I am still not certain this stream should stay open.  Let me fix it on my side